site/ja/index.html

```diff
@@ -523,13 +529,35 @@ <h2 class="section-heading">Eclipse Collectionsの歴史</h2>
             2015年、オープンソースプロジェクトとしての特徴を最大限に活かすため、GS CollectionsはEclipse Foundationに移管され、Eclipse Collectionsとして生まれ変わりました。
             コミュニティからのコントリビューションを受け付ける、完全にオープンなフレームワークになったのです。
         </p>
-        <hr class="section-heading-spacer">
+    </div>
+</div>
+<!-- History  -->
 
+<a class="anchor" name="refGuide"></a>
+<div class="content-section-a">
+    <div class="container-fluid">
+        <h2 class="section-heading">リファレンスガイド</h2>
+        <p class="lead">
+            ライブラリの詳細なリファレンスガイドはこちら:
+            <a href="https://github.com/eclipse/eclipse-collections/blob/master/docs/guide.md#eclipse-collections-reference-guide">Reference Guide</a>.
+        </p>
     </div>
     <!-- /.container -->
 
 </div>
-<!-- Introduction  -->
+<!-- Reference Guide  -->
+
+<a class="anchor" name="sourceCode"></a>
+<div class="content-section-b">
+    <div class="container-fluid">
+        <h2 class="section-heading">ソースコード</h2>
+        <p class="lead">
+            ソースコードはGitHubにホストされています: <a href="https://github.com/eclipse/eclipse-collections">Eclipse Collections</a></p>.
```

Comment:
the last `<\p>` tag is incorrect. That was one of the fixes I did for other files as a part of the commit you CP-ed.